src/views/HomeView.tsx

```diff
@@ -46,24 +47,24 @@ export const HomeView = () => {
         </Text>
         <AutoComplete
           searching={searching}
-          placeholder="Search data here"
+          placeholder="Nom de la base de données"
           width="100%"
           size="large"
           options={options}
           onSearch={searchHandler}
         />
 
         <Text h2 style={{ margin: '30px 0', textAlign: 'center' }}>
-          Categories
+          Catégories
         </Text>
 
         <div className="categories">
-          {datasets.map(({ category_name, data }, i) => (
-            <div className="category" key={i}>
-              <Text h4>{category_name}</Text>
+          {datasets.map(({ category, data }, dataIndex) => (
+            <div className="category" key={dataIndex}>
+              <Text h4>{category}</Text>
               {data.map(({ name, path }, i) => (
```

Comment:
I'm sorry I may be explained this in a wrong way so take a look in the console we have an error says: **Warning: Encountered two children with the same key, `20`. Keys**
This is because we have two lists here one **datasets** inside it, we have another list **data** and every item in a list should have a unique key, so we should then use different keys for each list for 
```ts
datasets.map(({category, data}, datasetIndex) => (
     <div key={datasetIndex}>
          {data.map(({name, path}, dataIndex) => (
              <Text key={dataIndex}>...</Text>
         ))}
    </div>
))
```

Reply:
Oh.. I got.. Sorry. Not super familiar with React yet.. 😉